common/Resources/ti.internal/extensions/node/process.js

```diff
@@ -147,7 +147,9 @@ process.emitWarning = function (warning, options, code, ctor) { // eslint-disabl
 	}
 	this.emit('warning', warning);
 };
-process.env = {};
+process.env = {
+	// DEBUG: '*' // uncomment to make use of npm 'debug' module. TODO: have CLI pass along env vars in some file/property for dev builds?
```

Comment:
Would be pretty cool to pass environment settings! In our backend, we use https://github.com/motdotla/dotenv for handling environments. And for production builds, things like API keys could be managed by this.

Reply:
Ah, yeah that's a nice solution for production builds. Yeah I was trying to debug some low-level node-api compatibility issues in using mocha on our test suite and hacked in the `process.env.DEBUG` value here for it and then realized it'd be super-useful to hook our CLI up to generate some files that passed along the ENV for development builds so you could do `DEBUG=* ti build -p ios` and it just magically worked. I'll probably open a ticket and add that.